packages/router/src/grpc/view-protocol-server/balances.ts

```diff
@@ -28,16 +28,17 @@ import { Amount } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/nu
 import { Base64Str, uint8ArrayToBase64 } from '@penumbra-zone/types/src/base64';
 import { addLoHi } from '@penumbra-zone/types/src/lo-hi';
 import { IndexedDbInterface } from '@penumbra-zone/types/src/indexed-db';
-import { getAssetId } from '@penumbra-zone/getters/src/metadata';
 import { multiplyAmountByNumber } from '@penumbra-zone/types/src/amount';
 import { Stringified } from '@penumbra-zone/types/src/jsonified';
 
 // Handles aggregating amounts and filtering by account number/asset id
 export const balances: Impl['balances'] = async function* (req, ctx) {
   const services = ctx.values.get(servicesCtx);
-  const { indexedDb } = await services.getWalletServices();
+  const { indexedDb, querier } = await services.getWalletServices();
 
-  const aggregator = new BalancesAggregator(ctx, indexedDb);
+  const latestBlockHeight = await querier.tendermint.latestBlockHeight();
```

Comment:
I think it's better to use `latestBlockHeight` rather than `fullSyncHeight` to avoid showing old prices during the sync process